versions/schematic-2.md

```diff
@@ -0,0 +1,235 @@
+**This specification is under development**
+# Sponge Schematic Specification
+
+**Version 2**
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt).
+
+## Introduction
+
+This specification defines a format which describes an area of a [Minecraft](http://minecraft.net) world for the purpose of serialization to disk and deserialization from disk. It is designed in order to allow maximum cross-compatibility between platforms, versions, and various states of modification.
+
+## Revision History
+
+Version | Date | Notes
+---|:---:|---
+1 | 2016-08-23 | Initial Version
+2 | 2018-08-19 | More features, easier handling, see [Changelog](#changelog)
+
+## Specification
+
+### Format
+
+All field names in the specification are **case sensitive**.
+
+#### <a name="formFile"></a>File
+
+The structure described by this specification is persisted to disk using the [Named Binary Tag](http://minecraft.gamepedia.com/NBT_format) (NBT) format (also see [here](https://wiki.vg/NBT)). Before writing to disk the NBT data must be compressed using the [GZip](https://www.gnu.org/software/gzip/) data compression algorithm. The highly recommended file extension for files using this specification is `.schem` or `.spongeschem` (chosen so as to not conflict with the legacy `.schematic` format allowing easy distinction between the two).
+
+#### <a name="formId"></a>Identification string
```

Comment:
I think it's more an explanation of the used format with reference to its definition. The definition itself is within MC Wiki and Wiki.vg.
Anyway, why should a definition be a problem? I define the "Format", which has to be used for the "File".

Also i renamed the "Resource location" to "Identification string", because i didn't know the name "Resource location". If it really is used by this name in code like you say, rename it back... is this a sponge thingy? Never heard of it like that...